src/ResourceManager/Network/Commands.Network/ExpressRouteCrossConnection/CrossConnectionPeering/GetAzureRMExpressRouteCrossConnectionPeeringCommand.cs

```diff
@@ -0,0 +1,55 @@
+// 
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// 
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Network.Models;
+using System.Linq;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet(VerbsCommon.Get, "AzureRmExpressRouteCrossConnectionPeering"), OutputType(typeof(PSExpressRouteCrossConnectionPeering))]
+    public partial class GetAzureRmExpressRouteCrossConnectionPeering : NetworkBaseCmdlet
+    {
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true)]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
+            HelpMessage = "The cross connection")]
+        public PSExpressRouteCrossConnection ExpressRouteCrossConnection { get; set; }
+
+        public override void Execute()
+        {
+            base.Execute();
+            if (!string.IsNullOrEmpty(this.Name))
```

Comment:
`this.` does nothing in any of these statements (including all the ones below). It is not needed and can be safely removed.

Reply:
Done